modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/raft/ZonePartitionRaftListener.java

```diff
@@ -307,6 +321,10 @@ public void addTableProcessor(TablePartitionId tablePartitionId, RaftTableProces
                         currentCommitedConfiguration.lastAppliedIndex,
                         currentCommitedConfiguration.lastAppliedTerm
                 );
+
+                // TODO https://issues.apache.org/jira/browse/IGNITE-24517 propagate lease information from txnStateStorage to newly added
```

Comment:
Will be fixed by https://github.com/apache/ignite-3/pull/5237 as well

Reply:
Adjusted.